.github/workflows/tests.yml

```diff
@@ -40,22 +40,43 @@ jobs:
       with:
         go-version: '1.21.0'
 
-    - run: make install-dev
-    - run: make unit-tests
+    - name: Resolve Karapace version
+      run: echo KARAPACE_VERSION=4.1.1.dev44+gac20eeed.d20241205 >> $GITHUB_ENV
```

Comment:
Need a better way to resolve the version here, cannot do `make install` as now the `src` is copied into the dev image.